readme.md

```diff
@@ -524,6 +524,33 @@ test.only.serial(...);
 
 This means you can temporarily add `.skip` or `.only` at the end of a test or hook definition without having to make any other changes.
 
+### Test macros
+
+Additional arguments after the test function will be passed as an array to the test function. This is useful for creating reusable test macros.
```

Comment:
That may be a cleaner way of saying it, but I'm trying to introduce the term. Maybe "reusable macros", but I think it's good as is.